package.json

```diff
@@ -1,9 +1,21 @@
 {
   "name": "sentinel-ast",
-  "version": "1.0.6",
+  "version": "1.0.7",
   "description": "Sentinel automated security testing framework",
   "main": "index.js",
   "repository": "https://github.com/nintexplatform/sentinel",
+  "keywords": [
+    "sentinel",
+    "security",
+    "test",
+    "testing",
+    "docker",
```

Comment:
@petedv Nice to have these keywords as well `cucumber` `gherkin`

Reply:
Yup. Updated

Reply:
What about automated and automation

Reply:
What about automated and automation

Reply:
I'll add automated. The other is there.

Reply:
Nvm. I see it was merged. :)